### DISCUSSION TITLE
DMR sound directly to speakers?

### DISCUSSION THREAD

**Discussion by joshuisken** [crowbarz/aiopioneer#28] [General]
In Homeassistant the input select while playing from a DLNA server is automatically set to "47" for my VSX-S510.
When the TV is switched on it plays fine, over HDMI-ARC, since that is enabled.
I tried to find what input selection is needed when I want to keep the TV in standby and still have audio directly over the speakers.
Any suggestion?

**Comment by crowbarz** [comment]
[Converted to a discussion as it seems to be related to how the AVR is used, rather than a problem with aiopioneer or the HA integration.]

You may be able to figure out the appropriate input for your specific receiver by using the AVR remote/front panel to configure the AVR inputs to the desired state, and then examining the AVR HA entity attributes via Developer Tools > States to see which source is selected. 

Another user may have more ideas if you provide more detail about your desired setup, eg. is the TV or the AVR connecting to your DLNA server here? I don't use ARC but I thought ARC transmits audio from the TV back to the AVR, but I doubt that is possible if your TV is in standby mode. So you might need to connect the AVR to the DLNA server instead, if the AVR supports it.
